src/custom/hooks/useRecentActivity.ts

```diff
@@ -143,9 +143,9 @@ function createActivityDescriptor(tx?: EnhancedTransactionDetails, order?: Order
     isPending = order.status === OrderStatus.PENDING
     isPresignaturePending = order.status === OrderStatus.PRESIGNATURE_PENDING
     isConfirmed = !isPending && order.status === OrderStatus.FULFILLED
-    isCancelling = (order.isCancelling || false) && isPending
-    isCancelled = !isConfirmed && order.status === OrderStatus.CANCELLED
     isCreating = order.status === OrderStatus.CREATING
+    isCancelling = (order.isCancelling || false) && (isPending || isCreating)
```

Comment:
nit, why is this? (the `isPending || isCreating` part?)

Maybe is more sef explanatory if you introduce a const for `isPending || isCreating` with a good name explaining what you try to model.

Reply:
Feel like another var will be too much.
Will add a comment to be explicit

Reply:
Take a look at [f7af233](https://github.com/cowprotocol/cowswap/pull/1698/commits/f7af23337536a68d33279f035dfb2905fe48e494)

Reply:
sounds good, it was just that i was wrapping my head around why is that. But a comment works